src/town_gui.cpp

```diff
@@ -1628,7 +1628,7 @@ static CargoTypes GetProducedCargoOfHouse(const HouseSpec *hs)
 
 struct BuildHouseWindow : public PickerWindow {
 	std::string house_info{};
-	bool house_protected = false;
+	static inline bool house_protected;
```

Comment:
A bit of an annoyance in what's accepted by C++, but all `this->house_protected` now don't follow our code style. They should be `BuildHouseWindow::house_protected` instead.